lib/bus.js

```diff
@@ -24,7 +26,16 @@ function connect() {
  */
 function disperseEvent(topic, payload) {
   try {
-    streams[topic].write(JSON.parse(payload));
+    let data = JSON.parse(payload),
+      scopedWrite = data.pid && data.hostname;
+
+    if (scopedWrite) { // We're dealing with amphora-redis-event-bus calls
+      if (data.pid === PID && HOSTNAME === data.hostname) {
+        streams[topic].write(data.msg);
+      }
+    } else {
+      streams[topic].write(JSON.parse(payload));
```

Comment:
could just pass `data` here instead of parsing again (cuz its already parsed on line 29)

Reply:
Whoops, good catch